aot/src/test/java/com/dylibso/chicory/imports/SpecV1ElemHostFuncs.java

```diff
@@ -0,0 +1,106 @@
+package com.dylibso.chicory.imports;
+
+import static com.dylibso.chicory.test.gen.SpecV1ElemTest.mInstance;
+import static com.dylibso.chicory.test.gen.SpecV1ElemTest.module1Instance;
+import static com.dylibso.chicory.test.gen.SpecV1ElemTest.module4Instance;
+
+import com.dylibso.chicory.runtime.GlobalInstance;
+import com.dylibso.chicory.runtime.HostGlobal;
+import com.dylibso.chicory.runtime.HostImports;
+import com.dylibso.chicory.runtime.HostTable;
+import com.dylibso.chicory.runtime.TableInstance;
+import com.dylibso.chicory.wasm.types.Limits;
+import com.dylibso.chicory.wasm.types.Table;
+import com.dylibso.chicory.wasm.types.Value;
+import com.dylibso.chicory.wasm.types.ValueType;
+
+public class SpecV1ElemHostFuncs {
```

Comment:
I assume that we are copy-pasting those HostFuncs from the `runtime-tests` module, correct?

If this is the case, we can probably refactor to avoid duplication (no need to do it in this PR though).

Reply:
Correct -- that was in the first version of AOT. I'll put up a PR to clean this up.